directory.json

```diff
@@ -13,6 +13,7 @@
 	"BP": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/India/BangPypers-api.json",
 	"CALLUG":"https://raw.githubusercontent.com/fossasia/fossasia-communities/master/India/LinuxUsersGroup-Calcutta-api.json",
 	"CDLug": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/China/ChengduGNU-LinuxUserGroupCDLug-api.json",
+	"Ceylon": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Sri%20Lanka/Ceylon-api.json",
```

Comment:
@mananwason I think @roonyh had mentioned in https://github.com/fossasia/directory.api.fossasia.net/pull/185 about Ceylon Linux not being an Open Source Organization.

Kindly cross-check before merging :)